l10n_us_form_1099/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Open Source Integrators <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is1099 = fields.Boolean('Is a 1099')
+    supplier = fields.Boolean(string='Is a Vendor',
+                              help="Check this box if\
```

Comment:
Please define one param per line as indicated in the multi-line fields declaration [example in the contribution](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models) (look in the code block slightly below the header I linked)

Additionally, the help string (if still multi-line), should break the string and continue again. Example:
```
x = fields.Char(
    help='one line'
         'another line',
)
```